tests/utils/queryApiGateway.test.ts

```diff
@@ -71,7 +71,7 @@ describe('Querying the API Gateway', () => {
       API_URL: goodUrl,
     })
 
-    const data = await queryApiGateway(goodRequest)
+    const data = await queryApiGateway(goodRequest, goodUniqueNumber)
```

Comment:
Note: For some weird reason, linting wasn't erroring on the fact thaht `queryApiGateway()` requires a second argument. Fixed in this PR.

Reply:
#444

Reply:
Thanks @lomky !